assets/css/main.css

```diff
@@ -243,7 +243,7 @@ body {
 
 button.clipboard {
   color: var(--main-color);
-  border-radius: 4px;
+  border-radius: 4px 0px 0px 4px;
```

Comment:
nice but can you do one thing remove the border radius I think it will look nicer with out border radious

Reply:
Yes. I thought you want that border radius. I removed please check.

https://deploy-preview-1043--clever-boyd-7dd108.netlify.app/documentation.html#toggleTheme